webapp-django/crashstats/crashstats/models.py

```diff
@@ -48,6 +51,8 @@ def config_from_configman():
         definition_source=[
             ElasticsearchConfig.required_config,
             PostgreSQLCrashStorage.required_config,
+            ReprocessingOneRabbitMQCrashStore.required_config,
```

Comment:
That's unfortunate. Doesn't appear to be a problem. SuperSearch works, the reprocessing works, and I can talk to postgres just fine. Maybe we're just lucky at the moment. 

The problem with referencing these is the way that config gets used. We essentially do this:
```python
implemenation = some_class(config_from_configman())
return implemenation.get(**request.GET)
```
Basically, we don't know if `some_class` is postgres, es or rabbitmq. 
We could mark up each class to tell which namespace to seek inside. E.g. instead of
```python
class SuperSearch(SocorroMiddleware):
    implemenation = socorro.external.es.supersearch.SuperSearch
```
We do
```python
class SuperSearch(SocorroMiddleware):
    implemenation = socorro.external.es.supersearch.SuperSearch
    implementation_namespace = 'es'
```